### DISCUSSION TITLE
Removal of mail server?

### DISCUSSION THREAD

**Discussion by viasux** [nextcloud/all-in-one#1258] [Questions] [answered]
I was unaware that there is a built in mail server. I would like to disable this functionality, as I don't need/want it. It also interferes with another mail server I have set up on my domain. Is there any way to remove this?

Is it left dormant if you never configure it as well?

**Comment by Zoey2936** [comment, marked as answer]
Nextdloud AIO has no inbuilt mail-server it just forces you to use an external smtp server

**Reply by viasux** [reply]
Gotcha, tysm for the help today!
